docker/scripts/setup

```diff
@@ -29,7 +29,7 @@ if [ -d "/scripts/cache" ]; then
   mv /scripts/cache vendor/
   chown -R huginn:huginn vendor/cache
 fi
-sudo -u huginn -H bundle install --deployment --without development test
+sudo -u huginn -H bundle install --deployment --with deployment
```

Comment:
Because if anyone uses this in production, we wouldn't want many possible dev tools since they can be security holes.